src/classes/UTIL_OrgTelemetry_SObject_BATCH.cls

```diff
@@ -74,10 +88,15 @@ public class UTIL_OrgTelemetry_SObject_BATCH implements Database.Batchable<SObje
      */
     public Database.QueryLocator start(Database.BatchableContext context) {
         switch on (jobMode) {
-            when RECURRING_DONATIONS {
+            when RECURRING_DONATIONS_OPPS {
```

Comment:
I switched the original job mode to use RECURRING_DONATIONS_OPPS since it was iterating over opportunities instead of Recurring donations

Reply:
Since this actually iterates over Opportunities, even though it's counting RD related stuff, it probably make sense to rename this enum to `OPPORTUNITIES`.